frontend/components/elements/ArticleRenderer.tsx

```diff
@@ -0,0 +1,21 @@
+import { RawComponent } from '../elements/RawComponent';
+import { ImageBlockComponent } from '../elements/ImageBlockComponent';
+import React from 'react';
+
+export const ArticleRenderer: React.SFC<{ elements: CAPIElement[] }> = ({
```

Comment:
Should this live elsewhere? I'm not sure, but part of me wonders if components should live under /components, but logic like this (which admittedly returns components, but doesn't define any) should live elsewhere. So as to more clearly separate out logic from the pure rendering stuff?

Reply:
I tend to agree with this. Perhaps we need a `/renderers` directory at the same level of `/components` and `/pages`? I'm assuming there will be different renderers for different pages.

Reply:
`AsyncClientComponent` lives in `components/lib`, could `ArticleRenderer` live there too?

Reply:
This makes sense :+1: